NICE.Identity.Management/ClientApp/src/components/App/App.tsx

```diff
@@ -26,28 +27,31 @@ export class App extends React.Component {
 			<Router>
 				<Header search={false} auth={auth} />
 
-				<Container
+				<main
```

Comment:
Could this be changed back to the Container component with the extra attribute (see suggestion), getting rid of the main tag.

```suggestion
				<Container
				        elementType="main"
				        role="main"
					id="content-start"
					aria-label="Start of content"
					aria-live="polite" />
```